.bundlemonrc

```diff
@@ -0,0 +1,78 @@
+{
+    "baseDir": "./build",
+    "files": [
+        {
+            "path": "1/mespapiers.<hash>.js"
```

Comment:
I think you can make this file smaller with:
`            "path": "<hash>/mespapiers.<hash>.js"`

Reply:
@JF-Cozy

Reply:
je teste également avec `"path": "*/mespapiers.<hash>.js"`